frontend/services/entities/mdm.ts

```diff
@@ -72,22 +113,70 @@ export default {
     return sendRequest("DELETE", MDM_PROFILE(profileId));
   },
 
-  getAggregateProfileStatuses: (teamId = APP_CONTEXT_NO_TEAM_ID) => {
+  // TODO: API INTEGRATION: we need to rework this when we create API call that
+  // will return the aggregate statuses for windows included in the response.
+  // Currently to get windows data included we will need to make a separate call.
+  // We will likely change this to go back to single "getProfileStatusSummary" API call.
+  getAggregateProfileStatuses: async (
+    teamId = APP_CONTEXT_NO_TEAM_ID,
+    // TODO: WINDOWS FEATURE FLAG: remove when we windows feature is released.
+    includeWindows: boolean
+  ) => {
+    // if we are not including windows we can just call the existing profile summary API
+    if (!includeWindows) {
+      return mdmService.getProfileStatusSummary(teamId);
+    }
+
+    // otherwise we have to make two calls and combine the results.
+    const [
+      profileStatuses,
+      diskEncryptionSummary,
+    ] = await mdmService.getAggregateProfileStatusesWithWindows(teamId);
+
+    return new Promise<ProfileStatusSummaryResponse>((resolve) => {
+      resolve(
+        generateCombinedProfileStatusSummary(
+          profileStatuses,
+          diskEncryptionSummary
+        )
+      );
+    });
+  },
+
+  getAggregateProfileStatusesWithWindows: async (teamId: number) => {
+    const [profileStatuses, diskEncryptionSummary] = await Promise.all([
+      mdmService.getProfileStatusSummary(teamId),
+      mdmService.getDiskEncryptionSummary(teamId),
+    ]);
+    return [profileStatuses, diskEncryptionSummary];
+  },
+
+  getProfileStatusSummary: (teamId = APP_CONTEXT_NO_TEAM_ID) => {
     const path = `${
       endpoints.MDM_PROFILES_AGGREGATE_STATUSES
     }?${buildQueryStringFromParams({ team_id: teamId })}`;
-
     return sendRequest("GET", path);
   },
 
-  getDiskEncryptionAggregate: (teamId?: number) => {
-    let { MDM_APPLE_DISK_ENCRYPTION_AGGREGATE: path } = endpoints;
+  getDiskEncryptionSummary: (teamId?: number) => {
+    let { MDM_DISK_ENCRYPTION_SUMMARY: path } = endpoints;
 
     if (teamId) {
       path = `${path}?${buildQueryStringFromParams({ team_id: teamId })}`;
     }
 
-    return sendRequest("GET", path);
+    // TODO: API INTEGRATION: change when API is implemented
+    return new Promise<IDiskEncryptionSummaryResponse>((resolve) => {
+      resolve({
+        verified: { macos: 0, windows: 5 },
+        verifying: { macos: 1, windows: 4 },
+        action_required: { macos: 2, windows: 3 },
+        enforcing: { macos: 3, windows: 2 },
+        failed: { macos: 4, windows: 1 },
+        removing_enforcement: { macos: 5, windows: 0 },
+      });
+    });
+    // return sendRequest("GET", path);
```

Comment:
Do we mean to leave the test data in?

Reply:
yes. API imlementation is not done yet. There will be another PR to fix up these integration points when those are done.